libsignal-service/src/sender.rs

```diff
@@ -634,30 +638,27 @@ where
             );
         }
 
-        // XXX maybe refactor this in a method, this is probably something we need on every call to
-        // get_sub_device_sessions.
         let mut sub_device_sessions = Vec::new();
         if let Some(uuid) = &recipient.uuid {
             sub_device_sessions.extend(
-                self.cipher
-                    .store_context
+                self.session_store
                     .get_sub_device_sessions(&uuid.to_string())?,
             );
         }
         if let Some(e164) = &recipient.e164() {
-            sub_device_sessions.extend(
-                self.cipher.store_context.get_sub_device_sessions(&e164)?,
-            );
+            sub_device_sessions
+                .extend(self.session_store.get_sub_device_sessions(&e164)?);
```

Comment:
`recipient.get_sub_device_sessions(session_store)` would be nice to have, which is what I meant with that comment.

Reply:
What do we do with this, @gferon ?